pyproject.toml

```diff
@@ -14,6 +14,8 @@ aw-client = {git = "https://github.com/ActivityWatch/aw-client.git"}
 python-xlib = {version = "^0.26", platform = "linux"}
 pypiwin32 = {version = "223", platform = "win32"}
 wmi = {version = "^1.4.9", platform = "win32"}
+pyobjc-framework-ApplicationServices = "^6.2"
+pyobjc-framework-CoreText = "^6.2"
```

Comment:
These need to have `platform` set, otherwise it will fail to install on other platforms.

Reply:
Good point.